### ISSUE TITLE
Script not working

### ISSUE THREAD

**Issue by MaxBobba** [1v/dislike-truck#24]
When I click "dislike", the script popup window returns a 403 error, mentioning that it won't connect because the developer didn't give me access to the app. What can I do?

**Comment by MaxBobba** [comment]
??

**Comment by 1v** [comment]
#17

**Comment by MaxBobba** [comment]
It's not a "quota" problem, please answer my issue clearly.

**Comment by 1v** [comment]
Did you follow guide https://github.com/1v/dislike-truck/wiki? If you do then you should get `message: "The request cannot be completed because you have exceeded your <a href="/youtube/v3/getting-started#quota">quota</a>."`

**Comment by MaxBobba** [comment]
No, I don't have this message... I got what I mentioned earlier!

**Comment by MaxBobba** [comment]
But I actually followed the guide

**Comment by Tamagz** [comment]
bro, how do u solve this problem ? do you solve this problem already ?

**Comment by Tamagz** [comment]
i followed the guide too from  https://github.com/1v/dislike-truck/wiki? but it didn't work too

**Comment by MaxBobba** [comment]
No, I'm waiting for the developer to answer...

**Comment by 1v** [comment]
@MaxBobba can you take screenshot of window you get?
Btw tested script with new api keys and it works though quota is still low.

**Comment by MaxBobba** [comment]
![df](https://user-images.githubusercontent.com/60177188/105061815-3bc1d200-5a7a-11eb-80ba-5ce77fe98e2a.png)

**Comment by 1v** [comment]
At OAuth consent screen there is Publishing status set it to production.

**Comment by MaxBobba** [comment]
Alright, did it and now I got another problem... it is not disliking videos. I verified the quotas screen, it says "queries per day: 10000". I don't know what the heck is happening now

**Comment by MaxBobba** [comment]
?? you there?

**Comment by 1v** [comment]
Don't know, just works for me. Was able to dislike ~200 videos before quota.

**Comment by 1v** [comment]
Script works. 

![image](https://user-images.githubusercontent.com/6566370/108930287-b45a1680-7656-11eb-8b1f-6db8cfcf0480.png)

**Comment by saulsmokeitall** [comment]
Hey bro I got a problem to but the thing is I actually have some coding experience so this is kinda getting me irritated lol so basically the script isn't fully executing I look in the web console and I find im on a chrome os, I will send a screen shot of my problem for your better understanding

**Comment by iWalk809** [comment]
help i click the dislike nothing happens i really need help
